packages/-ember-data/addon/setup-container.js

```diff
@@ -1,6 +1,7 @@
-import { deprecate } from '@ember/debug';
```

Comment:
The original import path is correct, we should not change it.

Can you explain what made you want to update?